packages/cxl-ui/scss/cxl-featured-image.scss

```diff
@@ -0,0 +1,25 @@
+:host {
+  position: relative;
+  display: block;
+  min-width: 360px;
+  height: 25vh;
+
+  #container {
+    position: absolute;
+    bottom: 0;
+    width: 100%;
+    height: 75%;
+    background-color: var(--cxl-brand-color-black, rgb(26, 26, 26));
```

Comment:
This needs using `var(--cxl-featured-image-bg-color)` which falls back to `--cxl-brand-color-black`. Different page types use different background color and it needs to be reflected here.